Fix actions events and performances

skjnldsv opened the pull request:
- Slow performances because all menus are hidden with v-show: switching to v-if
- We were using `$children`, so v-if don't display anything and since we create `this.actions` from `$children`, it was empty: now using `$slots.default` as the array is filled regardless of the template being shown or not.

skjnldsv commented:
> Gives a small performance boost: from ~2,1 to ~1,5 seconds respective from ~5,4 to 4,4 seconds (reference see [#486 (comment)](https://github.com/nextcloud/nextcloud-vue/pull/486#issuecomment-526935407)).

How do you calculate this?
It seems different from what I get

korelstar commented:
I start the vueexample app with the customized `App.vue` (200 items, see link above) on two different computers in Firefox. The time period is calculated from the difference of the timestamps of the console debug output (on update, updated).

skjnldsv commented:
Already an improvement!
So after some testing, this is not coming from the scoping at all.
I'll try to check the performances potential later :)

skjnldsv commented:
Btw, it's much more faster in production mode. :)
```
08:38:43.707 App.vue:74 vue: beforeUpdate
08:38:43.955 App.vue:90 vue: updated
08:38:43.958 App.vue:92 vue: updated -> nextTick
```

Though this is quite the amount of items, not sure where to start to optimise :thinking:
